webapp/app/partials/main/persona_buttons.html

```diff
@@ -1,7 +1,17 @@
-<a class="btn btn-view-nav btn-right-navbar nav-login-btn"
-   ng-if="user.loggedin"
-   ng-click="logout()">
-   <span>Logout</span></a>
+<span class="dropdown"
+      ng-if="user.loggedin">
+  <button id="logoutLabel" title="Logged in as: {{user.email}}" role="button"
+          href="#" data-toggle="dropdown" data-target="#"
+          class="btn btn-view-nav btn-right-navbar nav-repo-btn">
```

Comment:
nb. we've also used `nav-repo-btn` properties elsewhere eg. for the Infra menu. I plan to sort this out for all menus in a follow up bug.